autogpt_platform/backend/backend/integrations/webhooks/stripe.py

```
@@ -0,0 +1,183 @@
+            )
+        if age > STRIPE_TIMESTAMP_TOLERANCE:
+            raise HTTPException(
+                status_code=403, detail="Stripe webhook timestamp is too old"
+            )
+
+        # Sign the raw bytes: decoding and re-encoding the body would transcode
+        # the whole payload for nothing, and blow up on a non-UTF-8 body.
+        payload_body = await request.body()
+        expected = hmac.new(
+            signing_secret.encode("utf-8"),
+            msg=f"{timestamp}.".encode("utf-8") + payload_body,
```

Ruff (0.16.1)

[warning] Unnecessary UTF-8 `encoding` argument to `encode`

Remove unnecessary `encoding` argument

(UP012)